h/static/images/logo.svg

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg width="24px" height="28px" viewBox="0 0 24 28" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 3.8.3 (29802) - http://www.bohemiancoding.com/sketch -->
+    <title>Rectangle 2 Copy 17</title>
```

Comment:
I created a separate PR for this which minifies all SVGs in production builds https://github.com/hypothesis/h/pull/3642

I think it is useful to have the unoptimized version in-tree for diff-ability.